lib/foreman_ansible/register.rb

```diff
@@ -24,7 +24,7 @@
                  :'api/v2/ansible_roles' => [:destroy, :obsolete] },
                :resource_type => 'AnsibleRole'
     permission :import_ansible_roles,
-               { :ansible_roles => [:import, :confirm_import],
+               { :ansible_roles => [:import, :confirm_import, :sync],
                  :'api/v2/ansible_roles' => [:import] },
```

Comment:
Should `:sync` be added into the array for `api/v2/ansible_roles` instead?

Reply:
right, done.